psydac/api/ast/utilities.py

```diff
@@ -89,6 +91,22 @@ def logical2physical(expr):
         return new_expr
     else:
         return expr
+
+def _get_name(atom):
+    atom_name = None
+    if isinstance( atom, ScalarTestFunction ):
+        atom_name = str(atom.name)
+
+    elif isinstance( atom, VectorTestFunction ):
+        atom_name = str(atom.name)
+
+    elif isinstance( atom, IndexedTestTrial ):
+        atom_name = str(atom.base.name)
+
+    else:
+        raise TypeError('> Wrong type')
+
+    return atom_name
         
 #==============================================================================
 def compute_atoms_expr(atomic_exprs, indices_quad, indices_test,
```

Comment:
@saidctb Please see my comment to function `compute_atoms_expr_field`. I think it is appropriate to add a docstring to this function as well.

Reply:
done